docker/docker-compose.yml

```diff
@@ -7,12 +7,29 @@ services:
       dockerfile: $PWD/Dockerfile
     ports:
       - "8080:8080"
+    env_file:
+      .env.develop
+    networks:
+    - back-tier
 
-  # minio1:
-  #   image: minio/minio
-  #   expose:
-  #     - "9000"
-  #   environment:
-  #     MINIO_ACCESS_KEY: minio
-  #     MINIO_SECRET_KEY: minio123
-  #   command: server http://minio1/data
+  minio1:
```

Comment:
in the official documentation the example is for multi nodes solution. As we want to use it locally ( or on a our machine ) it would make more sense to use just a volume and a single instance. This code is not production ready and is not meant to be deployed in prod ( that's why we want to switch through a factory easily )